website/docs/integrator-section/verifiable-document/did/wrapping-document.md

```diff
@@ -2,5 +2,52 @@
 title: Wrapping Documents
 sidebar_label: Wrapping Documents
 ---
+import WrapSingleDocument from "/src/reusables/_wrap-single-document.mdx";
```

Comment:
Please include comments on each of these mostly duplicated sections. The best case is that we can reuse them as components somehow with only the directories being changed. If not, please include in each of these files where else they are duplicated. 

You can consolidate the list of all the paths they are duplicated at and then just copy paste them into each of those files. This way if we ever need to make changes we will know what other files also need to be changed. 🙂

Just something like this will do:
```suggestion
import WrapSingleDocument from "/src/reusables/_wrap-single-document.mdx";

<!--
This file is mostly duplicated with the same content at the following locations:
1. docs/integrator-section/verifiable-document/did/wrapping-document.md
2. docs/integrator-section/verifiable-document/ethereum/wrapping-document.md
 -->
```

Reply:
wondering which will be an easy way to maintain it up to date. 
To find out the other places the content is duplicated, search for `import WrapSingleDocument` will show all the occurrences...
If the comments are added for each occurrence, when there's another duplication, then all the comments need manual updating.

Reply:
ah I'm referring to the files with duplicated content not the ones that import shared reusable components. For those no need to add any comment. Please refer to my example above.

I chose the import statement to add this comment as I wanted to show an example at the top of the file. But it is not related to the `WrapSingleDocument` but rather the Wrapping Documents section in its entirety.

Reply:
Thanks for your clarification. 
The MDX files with duplicated content are in the `src/reusables` folder.

Each time the reusable content is being duplicated, the `import VariableName` statement will be at the top, meanwhile another line `<VariableName />` will be added somewhere in the file. 

So I suggest adding a comment in each MDX file like this: 
`<!--Content in this file is duplicated into multiple files. Search for <VariableName /> to locate all occurrences-->`

This will make it possible to search and find all occurrences.

Reply:
added this comment into the current two MDX files in `src/reusables`

Reply:
Apologies for the confusion. 
Now the steps for wrapping documents in batch are in a MDX file. 
Comments are also added into the DID and Ethereum respective files.